.github/workflows/aks.yaml

```diff
@@ -35,64 +36,103 @@ jobs:
       - name: Build and install cilium CLI binary
         run: sudo make install
 
-      - name: Azure Login
+      - name: Login to Azure
         uses: azure/login@77f1b2e3fb80c0e8645114159d17008b8a2e475a
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
 
-      - name: Azure CLI info
+      - name: Display Azure CLI info
         uses: azure/CLI@4b58c946a0f48d82cc2b6e31c0d15a6604859554
         with:
           azcliversion: 2.0.72
           inlineScript: |
             az account show
 
+      - name: Set up job variables
+        id: vars
+        run: |
+          if [ ${{ github.event.issue.pull_request || github.event.pull_request }} ]; then
+            PR_API_JSON=$(curl \
+              -H "Accept: application/vnd.github.v3+json" \
+              -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+              ${{ github.event.issue.pull_request.url || github.event.pull_request.url }})
+            SHA=$(echo "$PR_API_JSON" | jq -r ".head.sha")
+            OWNER=$(echo "$PR_API_JSON" | jq -r ".number")
+          else
+            SHA=${{ github.sha }}
+            OWNER=${{ github.sha }}
+          fi
+
+          echo ::set-output name=sha::${SHA}
```

Comment:
I don't see `sha` used in this file.

Reply:
Yea, since we don't have AKS tests running in-cluster... I decided to leave it there mostly for consistency, but we can remove it.

Reply:
I would remove it, since it just clutters the file (setting it uses several lines above), but it's mostly a nitpick.

Reply:
Since it's the only workflow that would diverge from the norm, I would probably leave it like that but can do either way 🤔 We're supposed to fix the reliance on the SP anyway (#203), so when it's gone it'll rejoin the others.